### DISCUSSION TITLE
NAME Tag - Are "/" required even if the individual does not have a surname?

### DISCUSSION THREAD

**Discussion by Norwegian-Sardines** [FamilySearch/GEDCOM#433] [Q&A] [answered]
Historically, in many cultures an inheritable (_using the g7:SURN, A family name passed on or used by members of a family_) surname was not needed.  Even today Iceland does not have a surname tradition. 

For individuals who do not have a surname, what is the prescribed notation for individuals who do not have a surname vs individuals who have a surname but the name is not known?

In my application "/" is not required and only used when:
1) a surname is provided (surrounding the surname with "/")  Example: John /Doe/
2) a surname is unknown (Empty "//") Example: John //

Individuals without surnames **do not** have an empty "//" !   Example: Rigborg Danielsdatter

I don't see any reference to this is the documentation, in particular that "/" are not required when a individual does not have a surname.

**Comment by dthaler** [comment, marked as answer]
Section 2.8 (Personal Name) has:
```
PersonalName = nameStr
             / [nameStr] "/" [nameStr] "/" [nameStr]
```
The first line of the above ABNF indicates that "/" are not required.

**Comment by Norwegian-Sardines** [comment]
Maybe I'm too stupid to understand this but does the first character in the following mean "or"?

/ [nameStr] "/" [nameStr] "/" [nameStr]

**Comment by dthaler** [comment]
Correct, section 1.1 (Characters) says:

> Character-level grammars are specified in this document using Augmented Backus-Naur Form (ABNF) as defined in [STD 68](https://www.rfc-editor.org/info/std68) and modified in [RFC 7405](https://www.rfc-editor.org/info/rfc7405). We use the term “production” to refer to an ABNF rule, supported by any other rules it references.

>
> The following is a brief summary of the parts of ABNF, as defined by STD 68 and RFC 7405, that are used in this document:

> * A rule consists of a rulename, an equals sign =, and 1 or more alternative matches.
> * **Alternatives are separated by slashes /.**

**Comment by Norwegian-Sardines** [comment]
Thanks, I'm used to a different notation for "or"
PersonalName = nameStr | [nameStr] "/" [nameStr] "/" [nameStr] 

I'm showing my age!
